Makefile.am

```diff
@@ -10,7 +10,7 @@ SUBDIRS = src
 DIST_SUBDIRS = examples $(SUBDIRS)
 
 man1_MANS = doc/man/man1/sasutil.1
-EXTRA_DIST = $(man1_MANS) doc
```

Comment:
If you remove this part, _make install_ will fail on machines that do not have _doxygen_ installed, because the file _doc/man/man1/sasutil.1_ will not be present in the tarball.

I don't know if this is actually a problem, though. :)